mne/preprocessing/ica.py

```diff
@@ -313,9 +313,9 @@ def __init__(self, n_components=None, max_pca_components=None,
                  "in 0.19. If you want to use Extended Infomax, specify "
                  "method='infomax' together with "
                  "fit_params=dict(extended=True).", DeprecationWarning)
-        if not check_version('sklearn', '0.15'):
-            raise RuntimeError('the scikit-learn package (version >= 0.15) '
-                               'is required for ICA')
+        if method == 'infomax' and not check_version('sklearn', '0.15'):
```

Comment:
shouldn't this be `if method == 'fastica' and ...`?

Reply:
Of course.